mirrord/cli/src/execution.rs

```diff
@@ -189,6 +194,40 @@ impl MirrordExecution {
         })
     }
 
+    /// Some features don't make a lot of sense together with targetless agents, so if needed,
+    /// warn the user, if possible, disable unnecessary features.
+    ///
+    /// # Errors
+    ///
+    /// [`IncompatibleWithTargetless`] if both targetless and ephemeral agent.
+    fn check_config_for_targetless_agent(config: &mut LayerConfig) -> Result<()> {
+        if matches!(config.target.path, Target::Targetless) {
+            if config.feature.fs.is_active() {
+                // active by default, so we don't want to fail on this and force users to disable,
+                // just disable it for them.
+                warn!("Disabling file system operations because no target was specified and the agent is not going to read the filesystem of any existing pod.");
+                config.feature.fs = FsConfig {
+                    mode: FsModeConfig::Local,
```

Comment:
I think this needs to be `LocalWithOverrides` since some files are needed (`/etc/hostname`/`/etc/hosts`)

Reply:
I'm thinking maybe that's an overkill to have this override :\
If we want to leave it, I'd remove the warning and just mention in docs it's supported only with `LocalWithOverrides` unlike the different target modes which you can specify.

Reply:
ok, removed the fs check.